tests/test_embedding.py

```diff
@@ -1,24 +1,71 @@
 from conftest import get_metrics
-from pyriemann.embedding import Embedding
+import numpy as np
+from pyriemann.embedding import (Embedding,
+                                 RiemannLLE,
+                                 barycenter_weights,
+                                 riemann_lle)
```

Comment:
I removed it alltogether. I adjusted the code from sklearn without giving it too much attention, and I think the sklearn code is not very well described. It doesn't really give the null_space of the matrix. So I just moved the code to the `riemann_lle` function since it's not called anywhere else.